webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -2549,7 +2549,6 @@ const AdminDefinition: AdminDefinitionType = {
                             },
                             help_text_markdown: false,
                             isDisabled: it.not(it.userHasWritePermissionOnResource(RESOURCE_KEYS.SITE.POSTS)),
-                            isHidden: it.configIsFalse('FeatureFlags', 'PostPriority'),
```

Comment:
Seems that the feature flag was hiding the overall ability to enable the feature in the System Console and it was defaulted to `false` out of the box? I created a test server but this was available so I'm not sure where is this feature flag is being set.

Reply:
> Seems that the feature flag was hiding the overall ability to enable the feature in the System Console and it was defaulted to `false` out of the box?

Yes, by default, the settings were hidden in the System Console.

I'm not sure why the setting showed up on the test server. Maybe they use some default feature flags?